replicate/configmap/configmaps.go

```diff
@@ -191,6 +191,7 @@ func (r *Replicator) ReplicateObjectTo(sourceObj interface{}, target *v1.Namespa
 
 	sort.Strings(replicatedKeys)
 	resourceCopy.Name = source.Name
+	resourceCopy.Labels = source.Labels
```

Comment:
This way, `resourceCopy` and `source` will actually contain a reference to the same map. Better to actually [copy the map](https://stackoverflow.com/questions/23057785/how-to-copy-a-map).

Reply:
thanks updated :)